ql/src/test/queries/clientpositive/groupby_grouping_sets_pushdown1.q

```diff
@@ -4,16 +4,51 @@ CREATE TABLE T1(a STRING, b STRING, s BIGINT);
 INSERT OVERWRITE TABLE T1 VALUES ('aaa', 'bbb', 123456);
 
 EXPLAIN EXTENDED SELECT * FROM (
-SELECT a, b, sum(s)
```

Comment:
Any reason to change the existing test? Could we keep it as-is and add the new ones after that?

Reply:
Fine, I will keep the existing test as it is, and thanks a lot for the review!